reportportal/values.yaml

```diff
@@ -592,6 +592,8 @@ storage:
 ingress:
   enable: true
   hosts: []
+  # Application path as a prefix, should start from '/' and be without trailing '/'
```

Comment:
Could you add a definition for this value like others' definitions (e.g. @param ingress.path )?